src/bin/wd_copier.rs

```diff
@@ -12,10 +15,40 @@ struct Opt {
     dst: PathBuf,
 }
 
+fn get_dirty_bytes() -> u64 {
```

Comment:
based on running `cat /proc/meminfo`, I think this is actually kB rather than bytes

Reply:
The file is but the interface isn't; `/proc/meminfo` shows `Cached:          4905980 kB`, but `println!("{}", procfs::Meminfo::new().unwrap().cached);` shows `5037039616` which is apparently that value reported by bytes.

Also keep in mind it's not actually kB, it's KiB (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/s2-proc-meminfo) but it's left written as kB due to legacy concerns.

Reply:
Good info. Looks good